crawler/src/main/java/nl/fhict/happynews/crawler/crawler/QuoteCrawler.java

```diff
@@ -70,6 +70,7 @@ void crawl() {
         for (Quote quote : entity.getContents().getQuotes()) {
             Post toAdd = new Post();
             toAdd.setSource("Inspirational Quote");
+            toAdd.setSourceName("quote");
```

Comment:
At this point im not sure what the original idea was, but since none of the posts use both I figured I'd make an executive decision